scripts/helpers/alloyComponents.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default [
+  {
+    name: "Activity Collector",
+    value: "activitycollector",
+    checked: true,
+  },
+  {
+    name: "Audiences",
+    value: "audiences",
+    checked: true,
+  },
+  {
+    name: "Personalization",
+    value: "personalization",
+    checked: true,
+  },
+  {
+    name: "Event Merge",
+    value: "eventmerge",
+    checked: true,
+  },
+  {
+    name: "Decision Engine",
+    value: "decisioningengine",
+    checked: true,
+  },
+  {
+    name: "Machine Learning",
+    value: "machinelearning",
```

Comment:
We'll need to add Streaming Media and Legacy Media Analytics components here too.